Services/Diagnostics/DiagnosticsLogger.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Http;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Runtime;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Diagnostics
+{
+    public interface IDiagnosticsLogger
+    {
+        Task<IHttpResponse> LogServiceStartAsync(string message);
```

Comment:
all these methods return `IHttpResponse`. Is the response actually used anywhere, other than for testing?  If it's just for testing, we should remove it. In fact, a class calling a logger should not know how the logs are collected (in memory, file, network etc.).  Probably this class will soon collect events in memory and send them out in batch, so there won't be a distinct and immediate http response for every event logged.